src/common-ui/containers/IndexDropdown.tsx

```diff
@@ -478,7 +478,8 @@ class IndexDropdownContainer extends Component<Props, State> {
     ) => {
         if (
             this.canCreateTag() &&
-            this.state.focused === this.state.displayFilters.length
```

Comment:
It's not clear to me the distinction between `this.state.focused = -1` and `this.state.focused =0` both are used in the code in this class.